accurest-core/src/test/groovy/io/codearte/accurest/builder/MockMvcSpockMethodBuilderSpec.groovy

```diff
@@ -814,4 +815,29 @@ class MockMvcSpockMethodBuilderSpec extends Specification implements WireMockStu
 			!spockTest.contains("\\u041f")
 	}
 
+	@Issue('177')
+	def "should generate proper test code when having multiline body"() {
+		given:
+			GroovyDsl contractDsl = GroovyDsl.make {
+				request {
+					method "PUT"
+					url "/multiline"
+					body('''hello,
+World.''')
+				}
+				response {
+					status 200
+				}
+			}
+			MockMvcSpockMethodBodyBuilder builder = new MockMvcSpockMethodBodyBuilder(contractDsl)
+			BlockBuilder blockBuilder = new BlockBuilder(" ")
+		when:
+			builder.given(blockBuilder)
+			def spockTest = blockBuilder.toString()
+			new GroovyShell(this.class.classLoader).evaluate("""import static com.jayway.restassured.module.mockmvc.RestAssuredMockMvc.*
+$spockTest""")
+		then:
+			notThrown(CompilationFailedException)
```

Comment:
I don't know any quick way to enhance that. In the end it would be probably good to provide a dedicated support for executing/evaluating Accurest specifications instead of `GroovyShell`. Something like [EmbeddedSpecification](https://github.com/spockframework/spock/blob/master/spock-specs/src/test/groovy/org/spockframework/EmbeddedSpecification.groovy) in Spock.